### PR TITLE
Feat/keuken 28 see measures

### PR DESCRIPTION
**Pull request by StefanKool**

### CODE REVIEW

File: src/client/pages/farm/measures.vue [hunk 1]
```diff
@@ -1,3 +1,125 @@
...
+        :key="index">
+        <div
+          class="measure-select">
+          <div class="measure-select__button">
+            <div class="measure-select__header">
+              <strong>{{ measure.title }}</strong>
+              <div class="measure-select__caption">KIES</div>
+            </div>
+          </div>
+          <!--<div class="measure-select__parcels">
```
[the remark this answers is not in the document]
Comment (StefanKool) [reply src/client/pages/farm/measures.vue]: This is not part of this sprint and moved this to a new branch

File: src/client/pages/farm/measures.vue [hunk 2]
```diff
@@ -1,3 +1,125 @@
...
+            <span class="measure-select__label">PERCELEN:</span>
+            <div class="parcel-chip">36452735 <md-icon class="parcel-chip__icon">clear</md-icon></div>
+          </div>-->
+        </div>
+      </div>
+      <footer-bar>
+        <template>Kies een maatregel</template>
+        <!--<template>Klik op een perceel om deze maatregel toe te voegen</template>-->
```
[the remark this answers is not in the document]
Comment (StefanKool) [reply src/client/pages/farm/measures.vue]: This is not part of this sprint and moved this to a new branch

File: src/client/pages/farm/index.vue [hunk 3]
```diff
@@ -26,3 +26,7 @@ export default {
   },
 }
 </script>
+
+<style>
```
Comment (markomalis) [inline src/client/pages/farm/index.vue]: remove this

File: src/client/pages/farm/measures.vue [hunk 4]
```diff
@@ -1,3 +1,24 @@
 <template>
-  <h2>Measures</h2>
+  <div>
```
Comment (markomalis) [inline src/client/pages/farm/measures.vue]: why the extra div?

File: src/client/pages/farm/measures.vue [hunk 5]
```diff
@@ -1,3 +1,24 @@
...
+import { MeasureList } from '../../components'
+
+export default {
+  components: { MeasureList },
+  computed: {
+    ...mapState('measures', ['measures']),
+  },
+  methods: {
+    selectedMeasure (value) {
+      alert(value)
```
Comment (markomalis) [inline src/client/pages/farm/measures.vue]: ?

File: src/client/components/measure-list/measure-list.vue [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+    return {
+      selectedMeasure: '',
+    }
+  },
+  computed: {
+    ...mapState('measures', ['measures']),
+  },
+  methods: {
+    selectMeasure(value) {
+      this.$emit('clicked', value),
```
Comment (markomalis) [inline src/client/components/measure-list/measure-list.vue]: emit a more descriptive event, for example 'selectMeasure'

File: src/client/components/measure-list/measure-list.vue [hunk 7]
```diff
@@ -0,0 +1,94 @@
+<template>
+  <div>
+    <div
```
Comment (markomalis) [inline src/client/components/measure-list/measure-list.vue]: solve this extra div thingy with a template tag

**Pull request merged**